scripts/verifyVesting.js

```diff
@@ -0,0 +1,44 @@
+const hre = require('hardhat')
+const fs = require('fs')
+
+async function verifyVesting() {
+    const contracts = JSON.parse(
+        fs.readFileSync(`./contracts.json`, 'utf-8')
```

Comment:
This should also come from a different file. Maybe better naming for both will be required, so we can easily distinguish which is which